sqlglot/generator.py

```diff
@@ -1425,14 +1422,9 @@ def values_sql(self, expression: exp.Values) -> str:
 
             selects.append(exp.Select(expressions=row))
 
-        subquery_expression: exp.Select | exp.Union = selects[0]
-        if len(selects) > 1:
-            for select in selects[1:]:
-                subquery_expression = exp.union(
-                    subquery_expression, select, distinct=False, copy=False
-                )
-
-        return self.subquery_sql(subquery_expression.subquery(expression.alias, copy=False))
+        alias = f" AS {expression.alias}" if expression.alias else ""
```

Comment:
is this only done for non pretty?

Reply:
Nope - that's a good point though. I can make it so that this is only done for non-pretty.